source/blog/2018-07-13-the-ember-times-issue-55.md

```diff
@@ -0,0 +1,136 @@
+author: Kenneth Larsen, Amy Lam, Miguel Braga Gomes, Ryan Mark, Sivakumar Kailasam, Jessica Jordan, Alon Bukai
+tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2018
+alias : "blog/2018/07/13-the-ember-times-issue-55.html"
+responsive: true
+---
+
+nuqneH Emberistas! 🐹
+
+Read either on the [Ember blog](https://www.emberjs.com/blog/2018/07/13/the-ember-times-issue-55.html) or in our [e-mail newsletter](https://the-emberjs-times.ongoodbits.com/2018/07/13/the-ember-times-issue-55) what has been going on in Emberland this week.
+
+This week you can learn about **updating** your ember-cli app 🖳. Learn from firsthand experience how to become an **addon maintainer** 💪. Get some tips on writing your own **RFCs** ✨. Check out the new **SEO tactics** of Ember 🔎 and last but not least learn about the potential move of Ember to a new **chat platform** 💬. Go ahead and enjoy!
+
+---
+
+## [Don't Worry, Ember CLI Got You Covered](https://github.com/ember-cli/ember-cli-update) 🖳
+The **number one** tool for updating Ember.js apps or addons just got **even better**. The [newest version](https://github.com/ember-cli/ember-cli-update/releases) of `ember-cli-update` now runs `qunit-dom-codemod` for you. This means that you, with close to no effort at all, can utilize this great addon for your tests.
+
+And the cool things is that `ember-cli-update` fetches new codemods **during runtime** - so no need to update to get this nice codemod! To run the codemod just type `ember-cli-update --run-codemods` and magic will take care of the rest for you.
+
+You can visit the addon page for [ember-cli-update](https://github.com/ember-cli/ember-cli-update) to read all about it or visit [qunit-dom](https://github.com/simplabs/qunit-dom) to learn about all the nice features it provides.
+
+---
+
+## [Start Small - How I Became an Ember Addon Maintainer by Ryan Mark](github.com/tylerturdenpants) 💪
+
+[Ember User Activity](https://github.com/elwayman02/ember-user-activity/) is a great addon.  It fit my needs and worked perfectly...until I needed to use **Fastboot**. Seeing how others had solved Fastboot related compatibility issues, I cloned [`ember-user-activity`](https://github.com/elwayman02/ember-user-activity/) and got to work. After a few suggested tweaks by the author, my PR was merged, and a new version was released.  A few weeks later, a new version of ember-cli was released. I've always updated my own app, but I had never done an addon.  Excited about this new **challenge**, I updated [`ember-user-activity`](https://github.com/elwayman02/ember-user-activity/) to the latest ember-cli version. Moments later, after submitting my PR, the author sent me an invitation to **collaborate**. The author asked if I could **help** him maintain the addon, and I **enthusiastically** agreed. As of today, I have published a handful of releases. Even though I didn't create the addon, I had **ownership**, and it was exciting to be a part of the addon **community**.
```

Comment:
I think it would be cool to have the same capitalization for `ember-user-activity` throughout the section - maybe we can stick to the lowercase variation, as it has been used most often?

`ember-user-activity`